### PR TITLE
Development branch

### PR DESCRIPTION
**Pull request by vvkuzmych**

### CODE REVIEW

**Review by yaorlov — CHANGES_REQUESTED** [review]
Please run `rake` and check all tests.
There are some failures, e.g. `/about_array_assignment.rb:23`

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (yaorlov) [inline .idea/inspectionProfiles/Project_Default.xml]: Please remove this file from git and move it to gitignore

File: .idea/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (yaorlov) [inline .idea/vcs.xml]: Please remove this file from git and move it to gitignore

File: .path_progress [hunk 3]
```diff
@@ -0,0 +1 @@
+26,34,34,38,41,49,61,62,83,96,152,152,153,160,160,160,160,160,160,152,152,152,153,153,153,153,154,154,154,154
```
Comment (yaorlov) [inline .path_progress]: Please remove this file from git and move it to gitignore

File: .idea/workspace.xml [hunk 4]
```diff
@@ -1,15 +1,540 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (yaorlov) [inline .idea/workspace.xml]: Please remove this file from git and move it to gitignore

File: .idea/inspectionProfiles/Project_Default.xml (hunk 1 above)
Comment (vvkuzmych) [reply .idea/inspectionProfiles/Project_Default.xml]: move file to gitignore

File: .idea/vcs.xml (hunk 2 above)
Comment (vvkuzmych) [reply .idea/vcs.xml]: move file to gitignore

File: .path_progress (hunk 3 above)
Comment (vvkuzmych) [reply .path_progress]: move file to gitignore

File: .idea/workspace.xml (hunk 4 above)
Comment (vvkuzmych) [reply .idea/workspace.xml]: move file to gitignore

**Review by yaorlov — APPROVED** [review]

**Pull request merged**